### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by pasqLisena** [DOREMUS-ANR/doremus-ontology#29]
We should start to add all the [documentation](https://docs.google.com/document/d/1BTA1WGaEfhcnS-45e2RJ77dZiQDBahDHiqNgx4xl1Lw) to the ttl file.

This includes also the translation in English of the documentation

**Comment by delahousse** [comment]
yes, it is planned. We wait for the translation of the doc to start this enrichment of the owl ontology.
